src/software/Localization/main_SfM_Localization.cpp

```diff
@@ -22,6 +22,17 @@ using namespace openMVG::sfm;
 
 #include <cstdlib>
 
+// Naive function for finding the biggest common root dir from two paths
+std::string FindCommonRootDir(const std::string & dir1, const std::string & dir2)
```

Comment:
You're right, there is not such function.
There is only the function to compare pathes between platforms stlplus::path_compare